dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractGistReadOnlyController.java

```diff
@@ -101,15 +100,15 @@
             .withFilter( new Filter( "id", Comparison.EQ, uid ) ) );
     }
 
-    @RequestMapping( value = "/gist", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE )
+    @GetMapping( value = "/gist", produces = MediaType.APPLICATION_JSON_VALUE )
     public @ResponseBody ResponseEntity<JsonNode> getObjectListGist(
         HttpServletRequest request, HttpServletResponse response )
     {
         return gistToJsonArrayResponse( request, createGistQuery( request, getEntityClass(), GistAutoType.S ),
             getSchema() );
     }
 
-    @RequestMapping( value = "/{uid}/{property}/gist", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE )
+    @GetMapping( value = "/{uid}/{property}/gist", produces = MediaType.APPLICATION_JSON_VALUE )
```

Comment:
`org.springframework.http.MediaType` is already imported, so we can remove `MediaType` here I think
Just need `produces = APPLICATION_JSON_VALUE`

Reply:
Tried it and when it would not compile realised this would require a `import static` for the constant.
I think we can try to do these imports but maybe we make this another PR that checks all controllers again.